pywinauto/application.py

```diff
@@ -1212,7 +1212,7 @@ def process_get_modules():
     # collect all the running processes
     pids = win32process.EnumProcesses()
     for pid in pids:
-        if pid != 0: # skip system process (0x00000000)
+        if pid != 0 and isinstance(pid, int): # skip system process (0x00000000)

```

Comment:
Just curious, how come that  ```win32process.EnumProcess``` returns non-integer values. And even if that happens ```process_module``` should raise a validation error on the bad ```pid``` parameter.
BTW, maybe it's better to filter out the system process before the loop:
```python
pids = list(win32process.EnumProcesses())
pids.remove(0)
for pid in pids:
    try:
         modules.append((pid, process_module(pid), None))
    except (win32gui.error, ProcessNotFoundError):
        continue
```